starter-code/index.js

```diff
@@ -1,10 +1,40 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
+    constructor(items, length) {
+      this.items = [];
+      this.length = 0;
+    }
+    add(item) {
+      this.items.push(item);
+      this.items.sort(function compare(a, b) {
+        if (a < b) return -1; 
+        if (a > b) return 1; 
+        if (a === b) return 0;
+      });
+      this.length++; 
+    }
+    get(pos) {
+      if (pos <= this.length) {
+        return this.items[pos];
+      }else {
+        throw new Error ("OutOfBounds");
+      }
+    }
```

Comment:
Ici, si tu as une length de 5, les éléments de ton array auront les indexes suivants : 0, 1, 2, 3, 4.
Donc si tu essayes de renvoyer l'élément à index 5, alors que ça ne va que jusqu'à 4, tu vas retourner undefined.
Il vaudrait donc mieux écrire : `if (pos < this.length)` :)